packages/http-client-csharp/eng/scripts/Generation.psm1

```diff
@@ -24,11 +24,15 @@ function Get-TspCommand {
     param (
         [string]$specFile,
         [string]$generationDir,
+        [string]$configFile = $null,
```

Comment:
it looks like everywhere is simply setting the configFile to the generationDir + "tspconfig.yaml".

Instead of passing this in we should be able to simply test if it exists and add it in this method without a new param.

Reply:
Hello @m-nash The configFile can be in any location, and yes, in our cadl-ranch test/testProject, it is in `packageDir`+`tspconfig.yaml`.   But for `Get-TspCommand` function, it is better to pass a parameter to provide the capability to set a config file in any location.

Reply:
I think its better to not have the duplication in N places.  This isn't a public script that needs to support all scenarios its a private script that we can adopt a specific convention for and in this case the tspconfig.yaml is always in the same spot.

Reply:
ok. I removed the `configFile` parameter.